pkg/manifests/types.go

```diff
@@ -121,6 +121,8 @@ type K8sPrometheusAdapter struct {
 
 // You can use the `DedicatedServiceMonitors` resource to configure dedicated
 // Service Monitors for the Prometheus Adapter
+// This setting is DEPRECATED and will be removed in a future version. In this
+// version this setting has no effect.
 type DedicatedServiceMonitors struct {
 	// When `enabled` is set to `true`, the Cluster Monitoring Operator (CMO)
```

Comment:
should we remove the whole comment and just reiterate that the setting is deprecated?

Reply:
Yes good point, makes a lot of sense. Guess we'll need Brians approval ultimately though.

Reply:
bike-shedding but we could leave the struct forever (or at least as long as we use the CMO configmap) but not document it. There's probably work involved as I don't think that the doc generation tool supports this easily.

Reply:
yeah that would also work. Maybe we add the deprecation notice for now and decide next release if we'll just drop it or make it invisible?